src/main/java/Parser.java

```diff
@@ -0,0 +1,37 @@
+public class Parser {
+
+    public static Command parse (String input) {
+        String[] inputSplit = input.split(" ");
+        if (input.equals("bye")) {
```

Comment:
Might be more appropriate to use switch-case statements here to organise the code and make the different commands more apparent to the readers!